vultr/resource_vultr_bare_metal_server.go

```diff
@@ -387,6 +394,14 @@ func resourceVultrBareMetalServerUpdate(ctx context.Context, d *schema.ResourceD
 		req.OsID = osID
 	}
 
+	if d.HasChange("mdisk_mode") {
```

Comment:
`ForceNew` means that no update will ever be necessary on this resource, since it will destroy and re-create.  So this whole code-block is unecessary.

Reply:
This has been removed.